diag_manager/README.md

```diff
@@ -59,27 +59,27 @@ ESM4_piControl
 ```
 
 ### 2.2 File Section
-The files are listed under the diagFiles section as a dashed array. 
+The files are listed under the diagFiles section as a dashed array.
 
 Below are the **required** keys needed to define each file.
-- **file_name** is a string that defines the name of the file. Do not add ".nc" and "tileX" to the filename as this will handle by FMS. 
-- **freq** is an integer that defines the frequency that data will be written. The acceptable values are:
-  - =-1: output at the end of the run only 
-  - =0: output every timestep 
-  - \>0: output frequency
-- **freq_units** is a string that defines the units of the frequency from above. The acceptable values are seconds, minutes, hours, days, months, years. 
-- **time_units** is a string that defines units for time. The acceptable values are seconds, minutes, hours, days, months, years. 
+- **file_name** is a string that defines the name of the file. Do not add ".nc" and "tileX" to the filename as this will be handled by FMS.
+- **freq** is a string that defines the frequency and the units that data will be written
+  - The acceptable values for freq are:
+    - =-1: output at the end of the run only
+    - =0: output every timestep
+    - \>0: output frequency and units (with a space between the frequency number and units e.g 24 hours)
+  - The acceptable values for units are seconds, minutes, hours, days, months, years.
```

Comment:
If -1 and 0 don't need units, then this section should be a little bit clearer.  It should specifically include the units with the value and explicitly state that units are not needed for -1 and 0.  
```
  - The acceptable values for freq are:
     - =-1: output at the end of the run only
     - =0: output every timestep
     - >0 units: output frequency and units (with a space between the frequency number and units e.g 24 hours)
Values of -1 or 0 do not require units.
```